production/scrapers/arizona_vaccine_manual.R

```diff
@@ -0,0 +1,92 @@
+source("./R/generic_scraper.R")
+source("./R/utilities.R")
+
+arizona_vaccine_manual_pull <- function(x){
+    "1VhAAbzipvheVRG0UWKMLT6mCVQRMdV98lUUkk-PCYtQ" %>%
+        googlesheets4::read_sheet(sheet = "AZ Vaccine", 
+                                  col_types = "Dccc")
+}
+
+arizona_vaccine_manual_restruct <- function(x){
+    x %>%
+        filter(!is.na(Date)) %>% 
+        filter(Date == max(Date))
+}
+
+arizona_vaccine_manual_extract <- function(x, exp_date = Sys.Date()){
+    
+    error_on_date(first(x$Date), exp_date)
+    
+    check_names(x, c(
+        "Date",
+        "Name", 
+        "Res.Anchor.Pop", 
+        "Res.Pct.Completed")
+    )
+    
+    if (as.numeric(x$Res.Pct.Completed) > 1.0){
+        stop(str_c("Vaccination percentage ", x$Res.Pct.Completed, 
+                   " should be between 0 and 1."))
```

Comment:
Safeguard for when I inevitably type 72.52 instead of 0.7252 and we accidentally end up with a 700% vaccination rate.